docs/developer/developer-mode-installation.md

```diff
@@ -0,0 +1,142 @@
+---
+id: developer-mode-installation
+title: "Developer Mode Installation"
+sidebar_position: 1
+sidebar_label: Developer
+keywords:
+  - Harvester
+  - harvester
+  - Rancher
+  - rancher
+  - Developer Mode Installation
+Description: Developer mode (dev mode) is intended to be used for testing and development purposes.
+---
+
+# Developer Mode
+
+:::caution attention
+
+Developer mode is intended to be used for development and testing purposes. Usage of this mode in production environments is not supported.
+
+:::
+
+## Prerequisites
+
+- The node has passed the [host-check](https://raw.githubusercontent.com/harvester/harvester/master/hack/host-check.sh)
+- Helm 3 and Git are installed on your local machine.
+
+## Installation
+
+You can install Harvester on an [RKE2](https://docs.rke2.io/) cluster using the [Helm](https://helm.sh/) CLI. For more information about installing and configuring the Harvester Helm chart, see the [readme](https://github.com/harvester/harvester/blob/master/deploy/charts/harvester/README.md).
+
+1. Create an RKE2 configuration file.
+    ```bash
+    sudo mkdir -p /etc/rancher/rke2
+    cat <<EOF | sudo tee /etc/rancher/rke2/config.yaml
+    disable:
+    - rke2-snapshot-controller
+    - rke2-snapshot-controller-crd
+    - rke2-snapshot-validation-webhook
+    node-label:
+    - harvesterhci.io/managed=true
+    token: token
+    cni:
+    - multus
+    - canal
+    EOF
+    ```
+
+1. Install RKE2.
+    ```bash
+    curl -sfL https://get.rke2.io | sudo sh -
+    sudo systemctl enable rke2-server.service
+    sudo systemctl start rke2-server.service
+    ```
+
+1. Create a kubeconfig file.
+    ```bash
+    sudo chmod 644 /etc/rancher/rke2/rke2.yaml
+    mkdir -p ~/.kube
+    cp /etc/rancher/rke2/rke2.yaml ~/.kube/config
```

Comment:
This will make `rke2.yaml` global readable. I think it's better to just copy out the file and then change the owner of it, like the following:
```suggestion
    mkdir -p ~/.kube
    sudo cp /etc/rancher/rke2/rke2.yaml ~/.kube/config
    sudo chown $(id -u):$(id -g) ~/.kube/config
```
And helm won't complain about the file permission.